Dockerfile

```diff
@@ -32,11 +32,10 @@ RUN set -x \
  && /fossology/install/scripts/php-conf-fix.sh --overwrite \
  && /fossology/utils/fo-installdeps -e -y \
  && $_cleanup
-RUN curl -sS https://getcomposer.org/installer | php && \
-    mv composer.phar /usr/local/bin/composer
 
 ADD . .
 RUN chmod +x /fossology/docker-entrypoint.sh
+RUN make composer_download && make composer_install
```

Comment:
This change now forces one to download composer every time one change a single line in the source tree and builds the image. (e.g. in the development loop)

I do not like this change, all dependencies should be loaded before adding the source.